expression/expr_to_pb.go

```diff
@@ -182,6 +182,10 @@ func (pc PbConverter) columnToPBExpr(column *Column) *tipb.Expr {
 		return nil
 	}
 	switch column.GetType().Tp {
+	case mysql.TypeBit:
+		if !IsPushDownEnabled("bit", kv.TiKV) {
```

Comment:
consider using ast.TypeStr like:
```suggestion
		if !IsPushDownEnabled(ast.TypeStr(column.GetType().Tp), kv.TiKV) {
```